### PR TITLE
The Vendor's Rising Consciousness and spelling fix

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Allow for the blood vendor and Terra Experimental cell field charger to talk.

Add more sentences in preexisting product_slogan for vendors that already have TTS so that they don't repeat the same joke.

Fix some spelling.

## Why It's Good For The Game

TTS uses vendor's product_slogan instead of product_ads, which is why you don't see the armament vendor talking to you. Would make some funny RP situations.

Also, marines are not soldiers.

## Changelog

:cl:
add: blood vendor and Terra Experimental cell field charger can use TTS!
add: more sentences in preexisting product_slogan for vendors that already have TTS so that they don't repeat the same joke
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
This will probably be very spammy considering there are like 40 of these vendors in prep.

**Comment by Arctrooper8** [comment]
Aren't marines literally just soldiers that can swim. I don't get the reasoning behind removing a bunch of references to soldiers, it feels like a real tomaytoes tomahtoes argument.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -1048,6 +1047,7 @@
 	icon_deny = "sustenance-deny"
 	wrenchable = FALSE
 	isshared = TRUE
+	product_ads = "Standard Issue Marine food!;It's good for you, and not the worst thing in the world.;Just fucking eat it.;You should have joined the Air Force if you wanted better food.;1200 calories in just a few bite!;Get that tabaso sauce to make it tasty!;Try the cornbread.;Try the pizza.;Try the pasta.;Try the tofu, wimp.;Try the pork.; 9 Flavors of Protein!; You'll never guess the mystery flavor!"
```
Comment (coldironwarrior) [inline code/game/objects/machinery/vending/marine_vending.dm:1050]: bite --> bites in the "1200 calories" slogan

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -1190,7 +1188,7 @@
 	icon_state = "bloodvendor"
 	icon_vend = "bloodvendor-vend"
 	icon_deny = "bloodvendor-deny"
-	product_ads = "The best blood on the market!"
+	product_slogans = "The best blood on the market!;Totally came from an ethical source!;O plus is the universal donor, use it!; Prevent hypovolemic shock starting today!"
```
Comment (coldironwarrior) [inline code/game/objects/machinery/vending/marine_vending.dm:1191]: O negative is universal donor. depending on if this gets maptext or not, might be more useful not to spell it out and just have "O- is the universal donor, use it!"? that way the suggested blood type people see in maptext matches exactly the blood type listed in the vendor.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 3]
```diff
@@ -1220,12 +1218,12 @@
 
 /obj/machinery/vending/armor_supply
 	name = "\improper Surplus Armor Equipment Vendor"
-	desc = "Am automated equipment rack hooked up to a colossal storage of armor and accessories. Nanotrasen designed a new vendor that utilize bluespace technology to send surplus equipment from outer colonies' sweatshops to your hands! Be grateful."
+	desc = "An automated equipment rack hooked up to a colossal storage of armor and accessories. Nanotrasen designed a new vendor that utilize bluespace technology to send surplus equipment from outer colonies' sweatshops to your hands! Be grateful."
```
Comment (coldironwarrior) [inline code/game/objects/machinery/vending/marine_vending.dm:1221]: utilize --> utilizes

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 4]
```diff
@@ -1354,12 +1352,12 @@
 
 /obj/machinery/vending/uniform_supply
 	name = "\improper Surplus Clothing Vendor"
-	desc = "Am automated equipment rack hooked up to a colossal storage of clothing and accessories. Nanotrasen designed a new vendor that utilize bluespace technology to send surplus equipment from outer colonies' sweatshops to your hands! Be grateful."
+	desc = "An automated equipment rack hooked up to a colossal storage of clothing and accessories. Nanotrasen designed a new vendor that utilize bluespace technology to send surplus equipment from outer colonies' sweatshops to your hands! Be grateful."
```
Comment (coldironwarrior) [inline code/game/objects/machinery/vending/marine_vending.dm:1355]: same as other, utilize --> utilizes

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 5]
```diff
@@ -1354,12 +1352,12 @@
...
 	icon_state = "surplus_clothes"
 	icon_vend = "surplus-vend"
 	icon_deny = "surplus_clothes-deny"
 	isshared = TRUE
-	product_ads = "If it moves, it's hostile!;How many enemies have you killed today?;Shoot first, perform autopsy later!;Your ammo is right here.;Guns!;Die, scumbag!;Don't shoot me bro!;Shoot them, bro.;Why not have a donut?"
+	product_ads = "Be the musician that you parents never approve you of. You gotta look good when you're in the battlefield.;We have all types of hats here!;What did one hat say to the other on the hiking trip? I'll wait here, you go on ahead;Sometimes, a beret is better than a helmet.;Drip is the priority, marine."
```
Comment (coldironwarrior) [inline code/game/objects/machinery/vending/marine_vending.dm:1360]: check first slogan. separate first ad from the second at "You gotta look good..."?

**Comment by TiviPlus** [comment]
Atomize lol

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 6]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/vending/weapon
-	name = "automated weapons rack"
-	desc = "A automated weapon rack hooked up to a colossal storage of standard-issue weapons."
+	name = "Automated weapons rack"
```
[the remark this answers is not in the document]
Comment (novaepee) [reply code/game/objects/machinery/vending/marine_vending.dm:2]: sad

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**